tests/integration/README.md

```diff
@@ -0,0 +1,65 @@
+# Tests
+
+## Overview
+
+This project contains smoke and integration UI tests for Busola.
+
+## Prerequisites
+
+Before testing, you need to copy your cluster's kubeconfig file to `fixtures/kubeconfig.yaml`.
```

Comment:
One small comment regarding this path. In the lighthouse's README there is written `Make sure your kubeconfig is copied as 'lighthouse/fixtures/kubeconfig.yaml'`. In the integration README, there is not `integration/` prefix.  Please choose one option for both paths, with or without the prefix.